### DISCUSSION TITLE
Request to add multiple languages

### DISCUSSION THREAD

**Discussion by LinLei0102** [sen-trie/nahidaQuest#1] [Ideas]
Thanks for developing such a fun placement game, Genshin fans are ecstatic!
I'm not sure if there's any consideration of getting multi-language?

**Comment by sen-trie** [comment]
Hmm, on paper, multi-lang seems possible - I could put all text strings and fstrings into a language object. At a quick glance, I think only the Wish letters, Aranara event popups and tutorial popups have text embedded in them (and so it would need a seperate process to edit in Photoshop). If somebody is willing enough to translate all the text in their own language, I do not mind adding such a feature.

**Reply by LinLei0102** [reply]
I can handle the images using Photoshop, I can do the translations, but there may be times when the translations are not correct. If the game gets a lot of play, someone should be able to come up with a better translation.
